src/node/rpc/frontend.h

```diff
@@ -90,9 +90,9 @@ namespace ccf
 
     void update_raft()
```

Comment:
As discussed, once we have implemented the [lightweight genesis model](https://github.com/microsoft/CCF/issues/178), we might not need to get the consensus from the store dynamically anymore but pass it to directly to the constructor of the frontend(s).